src/components/Tag/Tag.tsx

```diff
@@ -41,6 +42,24 @@ const borderStyle = (props: TagProps) => {
   };
 };
 
+const sizeCss = (size: TagProps['size']): NestedCSSProperties => {
+  if (size === 'DEFAULT') {
+    return {
+      padding: '0 20px',
+      lineHeight: '36px',
+      height: '36px',
+      fontSize: '0.8125rem',
+    };
+  }
+
+  return {
+    padding: '0 8px',
+    lineHeight: '22px',
+    height: '22px',
+    fontSize: '0.7125rem',
+  };
+};
+
```

Comment:
(size === 'DEFAULT') ? 
    return {
      padding: '0 20px',
      lineHeight: '36px',
      height: '36px',
      fontSize: '0.8125rem',
    } :
  return {
    padding: '0 8px',
    lineHeight: '22px',
    height: '22px',
    fontSize: '0.7125rem',
  };

Can we use the **ternary operator** here?. If conditions are untidy.